cafebabel/templates/home.html

```diff
@@ -24,19 +24,36 @@
 
 {% block body_class %}home{% endblock %}
 {% block body %}
-{% if articles %}
-  {{ card(articles|first) }}
-{% endif %}
+  <h2>Editor's pick</h2>
+  {% set article_editors_pick = articles_for_tag('editors-pick', limit=1) %}
+  {% if article_editors_pick %}
+    {{ card(article_editors_pick) }}
+  {% endif %}
 
   <h2>Sections</h2>
 {% for category in categories %}
   <a href={{ category.detail_url }} class="section-link {{ category.slug }}">{{ category.name }}</a>
 {% endfor %}
 
-{% if articles|length > 1 %}
+  <h2>Meet My Hood</h2>
+  {% set articles_meet_my_hood = articles_for_tag('meet-my-hood', limit=2) %}
+  {% for article_meet_my_hood in articles_meet_my_hood %}
+    {{ card(article_meet_my_hood) }}
+  {% endfor %}
+
+{% if articles %}
   <h2>Recently published</h2>
-  {% for article in articles[1:] %}
-    {{ card(article) }}
+  {% set article_network = articles_for_tag('network', limit=1) %}
+  {% for article in articles %}
+    {% if article not in (article_editors_pick, article_network) and article not in articles_meet_my_hood %}
+      {{ card(article) }}
+      {% if loop.index == 3 %}
```

Comment:
@cafebabel-webmaster can you update to reflect our latest discussion?

Reply:
hm yea. I will see how to do that.. 😄